osutil/cp_linux_test.go

```diff
@@ -36,11 +36,25 @@ func (s *cpSuite) TestCpMulti(c *C) {
 	c.Check(s.f2, testutil.FileEquals, s.data)
 }
 
+func (s *cpSuite) getReadOnlyFd(c *C) *os.File {
+	c.Check(osutil.CopyFile(s.f1, s.f2, osutil.CopyFlagDefault), IsNil)
+	c.Check(os.Chmod(s.f2, 0444), IsNil)
```

Comment:
small nitpick: these could be asserts since, if they fail, the subsequent checks don't tell us much
```suggestion
	c.Assert(osutil.CopyFile(s.f1, s.f2, osutil.CopyFlagDefault), IsNil)
	c.Assert(os.Chmod(s.f2, 0444), IsNil)
```